### ISSUE TITLE
Relax type checking when not using TypeScript?

### ISSUE THREAD

**Issue by pmcelhaney** [runem/lit-analyzer#160]
```js
const th = `<th aria-sort="${sortDirection}">header</th>`
```

The above code produces the following error

> Type 'string' is not assignable to '"ascending" | "descending" | "none" | "other"'

Without using TypeScript, I don't know how I could make the type any more specific than 'string'. 

According to the [docs](https://github.com/runem/lit-analyzer/blob/master/docs/readme/rules.md#-no-incompatible-type-binding):

> Assignments in your HTML are typed checked just like it would be in Typescript.

So the answer is clearly not "just use TypeScript". Can the rule be relaxed when the parser is not TypeScript? (We ought to be able to determine the parser by looking at `parserPath` or `getFilename()`on the [context object](https://eslint.org/docs/developer-guide/working-with-rules#the-context-object).)

**Comment by pmcelhaney** [comment]
As a work-around, I was able to suppress this warning by adding `<!-- @ts-ignore-->` to the HTML, per #1.
